### DISCUSSION TITLE
Index optimization for FULLTEXT?

### DISCUSSION THREAD

**Discussion by udlose** [olahallengren/sql-server-maintenance-solution#528] [Questions & Answers]
Is there any script for optimizing SQL FULLTEXT indexes?

**Comment by jeffchulg** [comment]
=> https://www.sqlshack.com/automatically-maintain-full-text-indexes-catalogs/

**Reply by udlose** [reply]
Thank you @jeffchulg!  I did come across that article in my research but was hoping/wondering if @olahallengren had anything for FULLTEXT.  All of the other work he did here was so comprehensive, I'd assumed FULLTEXT would have been covered somewhere here.
